samples/Backup/audit-vms-missing-backup-configurations/azurepolicy.json

```diff
@@ -0,0 +1,24 @@
+{
+  "properties": {
+    "mode": "All",
+    "parameters": {},
+    "displayName": "Audit VMs missing Backup Configurations",
+    "description": "Audit the VMs which are missing backup configurations in the environment",
+    "policyRule": {
+      "if": {
+        "field": "type",
+        "equals": "Microsoft.Compute/virtualMachines"
+      },
+      "then": {
+        "effect": "auditIfNotExists",
+        "details": {
+          "type": "Microsoft.RecoveryServices/backupprotecteditems",

```

Comment:
>backupprotecteditems [](start = 46, length = 20)

nit:backupProtectedItems

(naming convention is to use camelCase for resourceTypes)